specs/credentialmanagement/index.src.html

```diff
@@ -495,16 +512,17 @@ <h4 id="interfaces-credential-types-credential"><code>Credential</code></h4>
     interface Credential {
       readonly attribute DOMString id;
       readonly attribute DOMString type;
+      boolean isMatch (any filter);
```

Comment:
My intention is not about cro 

> I don't see how this "filtering" model fits with the kinds of cross-origin, third-party data that it seems it's meant to enable.

It is not intended to enable cross-origin third-party data. The intention is an API that allows for custom credentials defined in script (as opposed to requiring API updates).